### PR TITLE
feature/EUI-3881-fix-bug

### PR DESCRIPTION
**Pull request by rayliangatos**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3881

### Change description ###
Fix bug that it can't switch case detail tabs after inserting prepended tabs from URL

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/shared/components/case-viewer/case-viewer.component.spec.ts [hunk 1]
```diff
@@ -1214,3 +1250,119 @@ describe('CaseViewerComponent - prependedTabs', () => {
...
   });
 })
+
+// noinspection DuplicatedCode
+describe('CaseViewerComponent - Overview with prepended Tabs', () => {
+  let mockLocation: any;
+
+  let comp: CaseViewerComponent;
+  let f: ComponentFixture<CaseViewerComponent>;
+  let d: DebugElement;
```
Comment (connorpgpmcelroy) [inline src/shared/components/case-viewer/case-viewer.component.spec.ts:1260]: Minor change - Could we change f to component and d to element or similar please?

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Changes look good apart from one letter variables used

File: src/shared/components/case-viewer/case-viewer.component.spec.ts (hunk 1 above)
Comment (rayliangatos) [reply src/shared/components/case-viewer/case-viewer.component.spec.ts]: good shout I have copied paste it, now it's changing to something meaningful.

**Review by connorpgpmcelroy — APPROVED** [review]
Comments addressed and Sonar issues not relevant

**Pull request merged**